docs/release-notes/Percona-Everest-1.1.0-(2024-08-02).md

```diff
@@ -0,0 +1,127 @@
+ 
+
+    ???+ example "Example"
+        If you have created two schedules using backup storage bucket-1 and bucket-2, and an on-demand backup using backup storage bucket-3, you can only utilize one of these three backup storages to create the next on-demand backup or a schedule.
+
+
+    ![!image](../images/backup_storages.png)
+
+    ![!image](../images/pg_limitation_2.png)
+
+
+    ![!image](../images/on_demand_limitation_2.png)
+
+- You can only create up to three backup schedules for PostgreSQL.
+
+    ![!image](../images/max_three_schedules_pg.png)
+
+- You cannot change the storage location in existing schedules.
+
+- You cannot use the same storage location for multiple backup schedules.
+
+
+
+## Improvements
+
+- [EVEREST-1259](https://perconadev.atlassian.net/browse/EVEREST-1259) - We have set up a rate limiter to restrict the number of requests you can make to the API within a specific time frame. On the login page, an error will be thrown if you surpass this limit of requests within the given period.
+
+- [EVEREST-1134](https://perconadev.atlassian.net/browse/EVEREST-1134) -Starting with Percona Everest 1.1.0, you now have the convenience of upgrading the database version directly from the Namespaces page, without having to go through the hassle of using the edit DB wizard.
+
+- [EVEREST-1153](https://perconadev.atlassian.net/browse/EVEREST-1153) - We have revamped the CLI UX for install, upgrade, and uninstall commands. Prior to this, the screen was cluttered with logs that were hard to understand. But now, we have made this concise with loading animation and spinners.
+
+- [EVEREST-1088](https://perconadev.atlassian.net/browse/EVEREST-1088) -  We've removed the icons from the **Technology** column in the database list table to make it visually appealing.
+
+- [EVEREST-1196](https://perconadev.atlassian.net/browse/EVEREST-1196) - We've just added a confirmation dialog that pops up when you try to exit the wizard using the side navigation.
+
+
+- [EVEREST-1070](https://perconadev.atlassian.net/browse/EVEREST-1070) - We've updated the **restore** icon across Percona Everest for a consistent look.
+
+- [EVEREST-247](https://perconadev.atlassian.net/browse/EVEREST-247) - We've updated the Postgresql database icon on the UI for better clarity and visibility.
+
+### Backups and schedules
+
+- [EVEREST-1092](https://perconadev.atlassian.net/browse/EVEREST-1092) - To ensure a seamless user experience, you cannot initiate an on-demand backup while another backup is still in progress.
+
+- [EVEREST-1220](https://perconadev.atlassian.net/browse/EVEREST-1220) -  Starting with Percona Everest 1.1.0, you cannot use more than three different backup storages in total, including those used in existing backup schedules.
```

Comment:
Let's mention that this is only true for PostgreSQL and that the purpose of this limitation is to ensure that we can properly restore the backups.

Reply:
Done